docs/3.0.0-beta.x/getting-started/introduction.md

```diff
@@ -1,18 +1,80 @@
+Strapi, the most flexible open-source Headless CMS gives developers the freedom to use their favorite tools and frameworks while helping editors to easily manage their content and distribute anywhere. By making the admin panel and API easily customizable and extensible through a plugin system, Strapi enables the world’s largest companies to accelerate content delivery while building better digital experiences.
 
-Users love Strapi because it is open source, MIT licensed, fully customizable and based on Node.js. Strapi lets you manage your content and distribute it anywhere. Strapi allows you to securely and privately serve your database from your hosting and server of choice.
+::: tip Where the name come from?
+**Strapi** come from Boot**strap** your **API** 🎉<br>
+It's the origin purpose of the project.
+:::
 
-### Get Started
+## Features
 
-You are invited to get started using Strapi. You may explore Strapi by:
+### Custom content structure
 
-1. A [Quick Start Guide](quick-start.md) for more intermediate to advanced developers.
-2. A [Tutorial](quick-start-tutorial.md) for those who prefer a step-by-step introduction.
+With the admin panel of Strapi, you will be able to create the content structure that match your needs. Create it in just few clicks. Really fast to getting started.
 
-When you're done getting started, we invite you to join our [community](https://strapi.io/community).
+### Manage content
+
+The Strapi's admin panel provide you an intuitive interface to create edit and delete your content.
+
+### Developer-friendly API
+
+Your content have to be displayed on your devices, an API is required. Strapi provide you an API that match your need. Fetch the data you want via a REST API or via the GraphQL endpoint.
+
+### Roles & permissions
+
+A User system is built in Strapi, that let you manage how can access to what.
+
+### Plugin system
+
+You want a new feature in you application, fetch a plugin in the market place and tada! You can install a GraphQL endpoint in one command.
+
+### Customization
+
+You can customize every single pice of your application. The architecture of Strapi let you customize your app to match your exact needs.
+
+Don't stay stuck by the techno you choose.
+
+## The Strapi Ecosystem
+
+### Code Source
+
+Strapi is an Open Source project, all the source code is under the MIT licence. The core project is in the [strapi/strapi](https://github.com/strapi/strapi) GitHub repository and you will find all other tools into the [Strapi](https://github.com/strapi) GitHub organization.
+
+### Marketplace
+
+In the marketplace section of the admin panel, you will only found official Strapi plugins. For now, the Strapi marketplace is not open to everyone but will be soon, stay tuned.
+
+But you already can develop yours and publish them on npm.
+
+### Roadmap
+
+Strapi is a community oriented project. We are trying to be the more transparent as possible to let you have a vision on the future of the project.
```

Comment:
We are trying to be as transparent as possible so that every member of the community can get visibility on the future of the project.